Ports/openttd-opengfx/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=openttd-opengfx
+version=0.6.1
+workdir=.
+files="https://cdn.openttd.org/opengfx-releases/${version}/opengfx-${version}-all.zip opengfx-${version}-all.zip c694a112cd508d9c8fdad1b92bde05e7c48b14d66bad0c3999e443367437e37e"
+auth_type=sha256
+
+build() {
+    tar xf opengfx-${version}.tar
+}
+
+install() {
+    mkdir -p ${SERENITY_INSTALL_ROOT}/usr/local/share/games/openttd/baseset/
+    cp -a opengfx-${version}/* ${SERENITY_INSTALL_ROOT}/usr/local/share/games/openttd/baseset/
```

Comment:
```suggestion
    run_nocd mkdir -p ${SERENITY_INSTALL_ROOT}/usr/local/share/games/openttd/baseset/
    run_nocd cp -a opengfx-${version}/* ${SERENITY_INSTALL_ROOT}/usr/local/share/games/openttd/baseset/
```

Let's get ourselves some logging :)

Reply:
Done and done. :)